Modules/Project/Resources/views/subviews/create-project-details.blade.php

```diff
@@ -37,16 +35,21 @@
         </div>
         <div class="form-group col-md-5">
             <label for="total_estimated_hours">{{ __('Total Estimated Hours') }}</label>
-            <input type="number" class="form-control" name="total_estimated_hours" id="total_estimated_hours"
-                placeholder="Enter total estimated hours" value="{{ old('total_estimated_hours') }}">
+            <input type="number" class="form-control" name="total_estimated_hours" id="total_estimated_hours" placeholder="Enter total estimated hours" value="{{ old('total_estimated_hours') }}">
         </div>
         <div class="form-group offset-md-1 col-md-5">
             <label for="monthly_estimated_hours">{{ __('Monthly Estimated Hours') }}</label>
-            <input type="number" class="form-control" name="monthly_estimated_hours" id="monthly_estimated_hours"
-                placeholder="Enter monthly estimated hours" value="{{ old('monthly_estimated_hours') }}">
+            <input type="number" class="form-control" name="monthly_estimated_hours" id="monthly_estimated_hours" placeholder="Enter monthly estimated hours" value="{{ old('monthly_estimated_hours') }}">
+        </div>
+        <div class="form-group col-md-5">
+            <label for="contract_file">Upload File</label>
+            <div class="custom-file mb-3">
+                <input type="file" id="contract_file" name="contract_file" class="custom-file-input" multiple>
```

Comment:
what does `multiple` mean here?
```suggestion
                <input type="file" id="contract_file" name="contract_file" class="custom-file-input" multiple>
```

Reply:
We don't need `multiple` here. By the way, we had changed it in the pull request of the next part.